MAUI/Scheduler/accessibility.md

```diff
@@ -0,0 +1,231 @@
+---
+layout: post
+title: Accessibility in .NET MAUI SfScheduler control | Syncfusion
+description: Learn here about Accessibility support with the Syncfusion .NET MAUI Scheduler (SfScheduler) control, its elements and more.
```

Comment:
description: Learn here about Accessibility support with the Syncfusion .NET MAUI Scheduler (SfScheduler) control, its elements, and more.